components/build/README.md

```diff
@@ -1,80 +1,93 @@
+```
 
-## Testing 
+## Tekton Results integration
 
-To validate the pipelines are installed and working, you can run `./hack/build/m2-builds`  script which will build all the samples planned for milestone 2. 
+[Tekton Results](https://github.com/tektoncd/results) is installed in the cluster. Helper script `hack/build/set-tkn-results.sh` is provided to set configuration of for `tkn results` command.
 
-To deploy all the builds as they complete, add the `-deploy` option.
 ```
-./hack/build/m2-builds -deploy
+# ./hack/build/set-tkn-results.sh
+Configuration written to /home/myuser/.config/tkn/results.yaml
 
+Try it: tkn results list default
 ```
-You can also run the noop build `hack/build/quick-noop-build.sh`, that executes in couple seconds to validate a working install.
- 
-## Other Build utilties 
 
-The build type is identified via temporary hack until the Component Detection Query is available which maps files in your git repo to known build types. See `hack/build/repo-to-pipeline.sh`  which will print the repo name and computed builder type.
+## Shared Resources
 
-The system will fill with builds and logs so a utility is provided to prune pipelines and cleanup the associated storage. This is for dev mode only and will be done autatically by App Studio builds.
-Use `hack/build/prune-builds.sh` for a single cleanup pass, and `hack/build/prune-builds-loop.sh` to run a continous loop to cleanup extra resources. 
+Shared Secrets are provided to be used by projects, secrets is defined in one project but can be used by other projects.
 
-Use `./hack/build/util/check-repo.sh` to test your what auto-detect build will return.  
-```
-./hack/build/check-repo.sh  https://github.com/jduimovich/single-java-app
-https://github.com/jduimovich/single-java-app   -> java-builder
-  
-```
-If you want to check all your repos to see which ones may build you can use this script. You need to set you github id `export MY_GITHUB_USER=your-username` and it will test your repo for buildable content.  
+Available secrets:
 
-```
-./hack/build/ls-all-my-repos.sh | xargs -n 1 ./hack/build/check-repo.sh
-```
\ No newline at end of file
+| Name | Source | Description | Access |
+| -- | -- | -- | -- |
+| test-team-snyk | test-teams-snyk secret in test-team namespace | Snyk token used by HACBS pipelines | users/serviceaccounts with edit role |
+| redhat-appstudio-user-workload | redhat-appstudio-user-workload secret in build-templates namespace | Quay secret allowing to push into default AppStudio repository | users/serviceaccounts with edit role |
+| redhat-appstudio-staginguser | redhat-appstudio-staginguser secret in build-templates namespace | Quay secret allowing to push into component repositories in redhat-appstudio org | `pipeline` service accounts defined in [shared-resources-components.yaml](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/shared-resources/shared-resources-components.yaml)|
```

Comment:
@Michkov I think the existing doc had a gap wrt `redhat-appstudio-registry-pull-secret` (I had pinged @sbose78 about that in his original doc update PR but it never got resolved).

In particular, both the the use of the shared secret as well as the bypassing the use of the shared secret if you define `redhat-appstudio-registry-pull-secret` in your namespace.

All part of PLNSRVCE-250, with https://github.com/redhat-appstudio/application-service/pull/116 and https://github.com/redhat-appstudio/build-service/pull/29

Can we add those details to the doc as part of your cleanup?

thanks

Reply:
@gabemontero added https://github.com/Michkov/infra-deployments/blob/cleanup/components/build/README.md#repository-secrets

Reply:
looks good thanks @Michkov